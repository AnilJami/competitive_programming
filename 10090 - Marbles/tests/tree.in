7
2 7 3
4 3 2 2 0
5 3 0 1 0 0
3 50 40
4 15 3 16 10
8 12 12 12 21 12 15 6 14
13 15 23 20 18 14 1 21 9 9 18 23 10 4
4 30 16
7 10 4 1 2 0 6 0
14 14 12 10 6 14 12 4 9 9 2 3 0 0 0
17 8 2 4 6 10 8 0 4 7 12 0 6 8 4 2 8 10
15 12 14 7 2 3 10 0 8 0 5 8 10 6 8 12
6 10 6
5 0 4 2 3 4
7 1 4 2 2 3 4 2
10 3 0 0 4 1 4 1 4 1 1
5 4 3 3 4 1
3 1 0 4
9 1 2 2 4 3 3 4 2 0
8 20 12
5 9 8 3 2 7
2 8 0
3 1 4 8
7 4 5 7 0 3 2 1
10 8 2 8 1 9 7 0 2 3 1
8 2 2 2 4 2 5 6 7
3 0 9 7
4 3 5 6 2
4 24 10
5 0 8 11 5 9
6 8 2 2 0 8 4
7 1 2 7 3 2 1 9
3 10 7 8
5 28 8
3 13 2 10
5 2 4 8 9 12
7 0 7 13 8 12 6 4
10 2 12 12 4 5 12 2 8 6 9
6 2 12 11 9 8 4