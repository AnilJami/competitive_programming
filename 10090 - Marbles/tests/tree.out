5
28
43
23
64
46
87
